<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XO</class>
 <widget class="QWidget" name="XO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>448</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button8">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>260</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>20</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>140</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>15</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>15</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>350</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>350</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="currentMessage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
